src/studio/src/designer/frontend/ux-editor/containers/Container.tsx

```diff
@@ -256,6 +254,10 @@ export class ContainerComponent extends React.Component<IContainerProps, IContai
     }
   }
 
+  public alreadyExists = (containers: any, newId: string): boolean => {
```

Comment:
Could we rename this to containerIdExists or something similar? 👨‍💻 alreadyExists could mean a lot of things